src/Stratis.Bitcoin.Features.PoA/Voting/VotingController.cs

```diff
@@ -13,17 +13,17 @@
 namespace Stratis.Bitcoin.Features.PoA.Voting
 {
     [Route("api/[controller]")]
-    public class VotingController : Controller
+    public abstract class VotingControllerBase : Controller
```

Comment:
Seems like these changes aren't directly related / possibly should be part of a future PR instead? Just a bit harder to review

Reply:
Yeah, was supposed to be in this PR but it was getting too big to review easily so I've stopped and decided to do it in the next one.